src/main/java/dk/kb/util/webservice/Service2ServiceRequest.java

```diff
@@ -206,5 +203,20 @@ public static String getOAuth2Token() {
          return (String) m.get(OAuthConstants.ACCESS_TOKEN_STRING); 
                   
      }
+     /**
+      * Map status codes to known exceptions. 
+      * Http 2xx codes should not be mapped here.
+      * 
+      */
+     private static ServiceException mapServiceException(int statusCode) {         
+         switch (statusCode) {                      
+         case 400:            
+            return new InvalidArgumentServiceException();            
+         case 404:
+             return new NotFoundServiceException();            
```

Comment:
In KB-util we have a NoContentServiceException for HTTP code 204, should we add this ass well, just to have all defined service exceptions mapped?